src/main/java/edu/tamu/cap/utility/TripleUtil.java

```diff
@@ -0,0 +1,68 @@
+package edu.tamu.cap.utility;
```

Comment:
Would it be better to make these methods of CAP's Triple?

```
public static Triple fromJenaTriple(org.apache.jena.graph.Triple) { … }

public org.apache.jena.graph.Triple toJenaTriple() { … }

```

Reply:
I was being conservative with my time and was uncertain whether that would become part of the object payload during transport to the front-end. Otherwise, I think it make sense to have that as part of the Cap Triple.